pkg/config/config_test.go

```diff
@@ -323,8 +323,8 @@ func TestNewCreateChannelTxFailure(t *testing.T) {
 				profile.Policies = nil
 				return profile
 			},
-			err: errors.New("failed to create default config template: failed to create new channel group: " +
-				"failed to add policies: no policies defined"),
+			err: errors.New("creating default config template: failed to add policies to " +
+				"channel group: no policies defined"),
```

Comment:
super nit: it's much easier to read and search these when they aren't split across lines.

Reply:
Fair enough, I've been splitting these on the suggestion by the linter.